### ISSUE TITLE
Foodplace™ survivor camp, the new hope for all the hungry souls in the apocalypse

### ISSUE THREAD

**Issue by Termineitor244** [CleverRaven/Cataclysm-DDA#41789]
# Is your feature request related to a problem? Please describe.
The True Foodperson lacks the ability to create a faction camp in the break room/roof that is his/her dwelling, why the need to move to a empty camp, a fire station or an evac shelter when your secret base is right there?!

# Describe the solution you'd like

- As a possible start location with an NPC, and a secure, useful place to keep as a base, the "Restaurant Roof" that is the FoodLair for the True Foodperson should be able to be converted into a faction camp with certain available upgrades, since it has a big amount of unused space in the outside of the break room (First floor) of the restaurant, and for expansions you have the ground floor and the roof (second floor). 

- The restaurant in the ground floor has a lot of space that you could adapt for certain expansions, like a garage for your Foodtruck© or a big living quarters for all of your Foodkid© sidekicks that you will recruit to follow you and revive again the glory of Floodplace™, delivering Foodplace delicious food to all the hungry survivors out there, as you should.

- A possible alternative or addition to this could be to help the NPC Foodperson to build such a base, and help in the development of a new Foodplace faction, delivering NPCs to the faction and helping with materials to remodel the restaurant, obtaining a secure place to obtain more Foodplace delicious food, and even order some and see it delivered in your house by the Foodtruck©!

TL;DR, The possibility to start a new faction camp in the break room of Foodplace, with several of the standard upgrades (Beds, smoker, etc.) and at least one available expansion in the ground floor, with a little additional expansion (Maybe for a communications tower?) in the roof. Also, a set of missions to create a faction for the NPC Foodperson.

# Describe alternatives you've considered
Make the poor Foodperson leave his FoodLair and start a new base somewhere else...

# Additional context
The FoodPlace restaurant is a place that always spawns in the same way, so it should be easier to make a faction camp for this location (In contrast with, for example, a house, that can spawn in several different ways).
